src/DotNetBridge/NativeDataView.cs

```diff
@@ -626,22 +569,7 @@ public virtual void Dispose()
                     Data = null;
                 }
 
-                public virtual IEnumerable<KeyValuePair<string, ColumnType>> GetMetadataTypes()
-                {
-                    return Enumerable.Empty<KeyValuePair<string, ColumnType>>();
-                }
-
-                public virtual ColumnType GetMetadataTypeOrNull(string kind)
-                {
-                    Contracts.AssertNonEmpty(kind);
-                    return null;
-                }
-
-                public virtual void GetMetadata<TValue>(string kind, ref TValue value)
-                {
-                    Contracts.AssertNonEmpty(kind);
-                    throw MetadataUtils.ExceptGetMetadata();
-                }
+                public virtual Schema.Metadata Metadata => null;
```

Comment:
>public virtual Schema.Metadata Metadata => null; [](start = 16, length = 48)

This information can be stored in `Schema[col].Metadata`.

Reply:
Replaced MetaData with Schema.DetachedColumn so this won't be needed.

---
In reply to: [250277017](https://github.com/Microsoft/NimbusML/pull/90#discussion_r250277017) [](ancestors = 250277017)